packages/webchat/src/store/composer.ts

```diff
@@ -52,7 +50,7 @@ class ComposerStore {
       this._sentHistoryIndex = 0
 
       if (this.rootStore.config.enablePersistHistory) {
-        window.BP_STORAGE?.set(SENT_HISTORY_KEY, takeRight(this._sentHistory, constants.SENT_HISTORY_SIZE))
+        window.BP_STORAGE.set(SENT_HISTORY_KEY, takeRight(this._sentHistory, constants.SENT_HISTORY_SIZE))
```

Comment:
BP_STORAGE is defined? Where is that created?

Reply:
It's kind of a hack, but I import it in the root index.tsx file. I think we should rethink the storage class and make something cleaner and simpler. Maybe in another PR tho?